src/app.js

```diff
@@ -1,21 +1,10 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const { terminal } = require('./terminal');
+const { moveFile } = require('./moveFile');
 
-module.exports = sum;
+terminal.question(
+  // eslint-disable-next-line max-len
+  'Enter the command to move the file to another directory.\nFor example, "mv file.txt ./someDir/" ',
```

Comment:
```suggestion
  'Enter the command to move the file to ' +
  'another directory.\nFor example, "mv file.txt ./someDir/" ',
```
you may split it in this way (not critical :))